recipes-kernel/linux/linux-nilrt.inc

```diff
@@ -41,6 +41,7 @@ SRCREV ?= "${AUTOREV}"
 PV = "${LINUX_VERSION}+git${SRCPV}"
 CVE_VERSION = "${KERNEL_VERSION}+git${SRCPV}"
 
+LICENSE = "GPL-2.0-only"
```

Comment:
`recipes-kernel/linux/linux-yocto.inc` that we include above already contains this `LICENSE` line so we shouldn't require it. I ran `bitbake -e linux-nilrt` to confirm this.
So I'm not sure why we'd need this change to generate spdx for modules. Maybe it's a bug in spdx generation code instead?

Reply:
Yeah, you're right. I did a clean-build, and the kernel-module-* packages get generated without my fix in the initramfs SPDX. I installed it on another newly formatted target and can see that this fix is not necessary. My python parser script had reported a discrepancy because of a mismatch in the commit hashes.

So it seems like the "fix" for the bug is more of a documentation issue where we inform them that the SBOM should be picked up from the initramfs as opposed to the safemode rootfs. Does that sound correct?